ingestfile

```diff
@@ -42,11 +43,12 @@ _usage(){
 
 user_input="${*}"
 OPTIND=1
-while getopts ":pienh" OPT ; do
+while getopts ":pienah" OPT ; do
```

Comment:
I suggest to have here the same order as below, because it’s easier to maintain.